client/my-sites/comments/comment/comment-html-editor.jsx

```diff
@@ -0,0 +1,242 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import { delay, each, filter, get, map, reduce } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import AddImageDialog from 'post-editor/editor-html-toolbar/add-image-dialog';
+import AddLinkDialog from 'post-editor/editor-html-toolbar/add-link-dialog';
```

Comment:
We're still +50% on comment bundle size. @Copons, as I mentioned before, let's ignore the dialogs for now, and simply insert a tag.

![screen shot 2018-02-15 at 11 01 18 am](https://user-images.githubusercontent.com/1270189/36275438-b1c0bd58-123f-11e8-84d5-f967f45f5f9c.png)

Reply:
The dialogs and even this toolbar component I think are good candidates to async load in. We can of course try to do that as part of follow up PRs

Reply:
Sure, you're right, I totally forgot about that removing the dialogs. 🤦‍♂️